bundles/org.openhab.binding.pushbullet/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
```

Comment:
This looks like an old .classpath file. Was this generated by the eclipse in the new buildsystem environment? (Also have a look at classpath files in other bundles)

Reply:
the classpath file was a copy from some other binding as I use IntelliJ instead of Eclipse. Could you point me to a "good" one?

Reply:
Perfect. @openhab/2-x-add-ons-maintainers Can we please remove all .project and .classpath files now finally. We have the first incidents of a non Eclipse contributor :)

Reply:
I don't see why we need to break everyone's workspace by this WIP PR? It's also no first because the Google TTS add-on a few issues for Eclipse users because it was made in IntelliJ.

Reply:
Because we have 3 official binding development IDEs now as seen in the official docs. If you say that we want to keep .project files I guess you are alright with also checking in .vscode directories and .idea files?

Reply:
ah, @wborn , does the classpath file as it is in the PR break everyone's workspace or would removing all .project and .classpaths overall break everything?

I just want to be sure to cause minimal disruption for everyone 😊

Reply:
Also, the PR is not really in a WIP state but actually usable.
I'll happily remove that label as soon as Jenkins manages to build things.

Reply:
The removal of all project files would break everyone's workspace. I'd add the removal to the OH3 TODO list so they are gone when the repo is shrunken (https://github.com/openhab/openhab2-addons/issues/5371). I usually don't commit Eclipse .project/.classpath files to Git on other projects. But they are here now because this repo has a different legacy.

Reply:
Added a nicer classpath file from a recently contributed binding (mail, to be precise)